inject/titus-nsenter.c

```diff
@@ -104,7 +106,7 @@ static int set_up_apparmor(char apparmor_profile[1024], int apparmor_fd) {
 
 	// we can use dprintf, but this just makes error handling cleaner
 	memset(writebuf, 0, sizeof(writebuf));
-	n = sprintf(writebuf, "changeprofile %s", apparmor_profile);
+	n = snprintf(writebuf, 1024, "changeprofile %s", apparmor_profile);
```

Comment:
Please do not use 1024, and instead sizeof(writebuf) - 1.

Reply:
@alekar this came from your cmake change commit.

Reply:
True. Also, writebuf needs to be larger.

```
prod titusagent-main01cell003-m5.metal.001-v002490 us-west-2 i-0516f98bdf9357b3f
(nfsuper) ~ $ sudo cat /sys/module/apparmor/parameters/path_max
8192
```

Reply:
Ill make a second review for this